wear/src/main/java/io/homeassistant/companion/android/home/views/SensorManagerUi.kt

```diff
@@ -49,21 +48,36 @@ fun SensorManagerUi(
                     ListHeader(id = sensorManager.name)
                 }
                 val currentSensors = allSensors?.filter { sensor ->
-                    availableSensors.firstOrNull { availableSensor ->
+                    allAvailSensors?.firstOrNull { availableSensor ->
                         sensor.id == availableSensor.id
                     } != null
                 }
 
-                items(availableSensors.size, { availableSensors[it].id }) { index ->
-                    val basicSensor = availableSensors[index]
-                    val sensor = currentSensors?.firstOrNull { sensor ->
-                        sensor.id == basicSensor.id
+                if (allAvailSensors?.isEmpty() == true) {
+                    item {
+                        Column(
+                            modifier = Modifier
+                                .fillMaxSize()
+                                .padding(vertical = 32.dp),
+                            horizontalAlignment = Alignment.CenterHorizontally,
+                            verticalArrangement = Arrangement.Center
+                        ) {
+                            ListHeader(id = R.string.loading)
+                            CircularProgressIndicator()
+                        }
+                    }
+                } else {
+                    items(allAvailSensors!!.size, { allAvailSensors[it].id }) { index ->
```

Comment:
`!!` could result in a crash as the `else` branch is also used when `allAvailSensors` is `null`